spec/readme.md

```diff
@@ -188,6 +188,43 @@ For a very consistent subway service, there may be one route for each direction.
 }
 ```
 
+### `stopover`
+
+A `stopover` represents a vehicle stopping at a stop/station at a specific time.
+
+```js
+{
+	type: 'stopover', // required
+	// - stop/station id or object
+	// - required
+	stop: '12345-678',
```

Comment:
I think we should insert new lines after every attribute to clarify that the comments describe the thing below them instead of above them (I got confused by that for a moment).

Reply:
We may also consider using a table instead.